get_local_image_transform_command.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2020–2021 CEA
+#
+# Author: Joël Chavas <joel.chavas@cea.fr>
+#
+# Licensed under the Apache Licence, Version 2.0 (the "Licence");
+# you may not use this file except in compliance with the Licence.
+# You may obtain a copy of the Licence at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the Licence is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the Licence for the specific language governing permissions and
+# limitations under the Licence.
+
+import sys
+import os
+
+import requests
+import argparse
+
+"""
+Source and target spaces can be one of the 4 spaces:
+"MNI 152 ICBM 2009c Nonlinear Asymmetric"
+"MNI Colin 27"
+"Big Brain (Histology)"
+"Infant Atlas"
+"""
+SOURCE_SPACE_DEFAULT = "MNI 152 ICBM 2009c Nonlinear Asymmetric"
+TARGET_SPACE_DEFAULT = "Big Brain (Histology)"
+
+
+SERVER_ADDRESS_DEFAULT = "http://0.0.0.0:8080"
```

Comment:
I would use `https://hbp-spatial-backend.apps.hbp.eu` as default, which will work as soon as the `master` branch is deployed